activity-metrics/issue-resolution-efficiency.md

```diff
@@ -1,25 +1,62 @@
 # Issue Resolution Efficiency
 
 ## 1. Description
-What is the number of closed issues/number of abandoned issues?
+What is the number of closed issues/number of open issues?
 
 ## 2. Use Cases
-Provide examples of how the metric might inform different stakeholders through use cases.
+* A project interested in knowing if they are coping well or not
+with the issues being open, and how that evolves over time.
+This metric informs them exactly about that.
+If there are more issues being opened that issues being closed
+during a long period of time, the number of unresolved issues is going to grow.
 
 ## 3. Formula
-A generic formula (in pseudo code) to generate the metric.
+The definition is for a time period (for example, for a certain month):
+
+**Definitions:**
+
+* `issues_closed`: Total number of issues closed during the period
+* `issues_opened`: Total number of issues opened during the period
+* `issues_backlog`: Total number of issues open at the beginning of the period (backlog of open issues)

```

Comment:
Does it make sense to have this be the beginning or the end? I think either one would generate the same number because you are going to simply be adding "open now, but not opened during this period" in effect. I am tryin to think of when the number would be different at the beginning or end if its a windowed time period?